cmd/skaffold/app/cmd/render.go

```diff
@@ -25,13 +25,17 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
 var (
-	showBuild        bool
-	renderOutputPath string
+	showBuild           bool
+	renderOutputPath    string
+	fromBuildOutputFile flags.BuildOutputFileFlag
```

Comment:
```suggestion
	buildOutputFile flags.BuildOutputFileFlag
```

Reply:
`var` blocks are global in Go, and there's already such a variable for the deploy command... I'll make both variable names unique `{deploy,render}FromBuildOutputFile` unless you have a better idea.

Reply:
`deployBuildOutputFile` and `renderBuildOutputFile` seem fine